modules/SofaGeneralEngine/MeshBarycentricMapperEngine.cpp

```diff
@@ -20,20 +20,14 @@
 * Contact information: contact@sofa-framework.org                             *
 ******************************************************************************/
 #define SOFA_COMPONENT_ENGINE_MESHBARYCENTRICMAPPERENGINE_CPP
+
 #include <SofaGeneralEngine/MeshBarycentricMapperEngine.inl>
 #include <sofa/core/ObjectFactory.h>
 #include <sofa/defaulttype/VecTypes.h>
 #include <sofa/defaulttype/RigidTypes.h>
 
-namespace sofa
-{
-
-namespace component
-{
-
-namespace engine
+namespace sofa::component::engine
```

Comment:
I like it, but didn't we say we'd keep clear of namespace concatenation & pragmas in sofa modules & only apply them to plugins?